app/ui/browser/model/index.js

```diff
@@ -66,6 +66,9 @@ export const UIState = Immutable.Record({
 
   // What the user has typed into the location bar, stored by page id
   userTypedLocation: Immutable.Map(),
+
+  // Should the autocomplete popup be visible
+  showCompletions: true,
```

Comment:
would we want this to be indexed by page? I'm not sure if we keep the completions box open when you switch tabs.